src/app/files/file-explorer.js

```diff
@@ -470,7 +470,8 @@ fileExplorer.prototype.toGist = function (id) {
     return data.files || []
   }
 
-  this.packageFiles(this.files, 'browser/gists/' + id, (error, packaged) => {
+  const folder = id ? 'browser/gists/' + id : 'browser/'
```

Comment:
would be nice to have the comment here

Reply:
"If 'id' is not defined, it is not a gist update but a creation. we have to take the file from the browser explorer."